scripts/issue-queries.js

```diff
@@ -0,0 +1,248 @@
+exports.Queries = Object.freeze({
```

Comment:
please move to new folder `dashboard`
this is a good practice Akshat started -> https://github.com/asyncapi/website/tree/master/scripts (`Tools`)
so helper files go to dedicated folders as it is getting messy